python/lsst/ts/mtaos/wavefront_collection.py

```diff
@@ -23,21 +23,26 @@
 
 import warnings
 from collections import deque
+import re
 
 import astropy.units as u
 import numpy as np
 from astropy.table import QTable
 
 
 class WavefrontCollection(object):
-    def __init__(self, maxLeng: int) -> None:
+    def __init__(self, zernike_column_pattern: str, maxLeng: int) -> None:
```

Comment:
I think instead of receiving a string and hosting the re here, I would suggest passing the `Patter` object.